src/Configuration/Cache.php

```diff
@@ -17,6 +17,7 @@
  * @author Fabien Potencier <fabien@symfony.com>
  * @Annotation
  */
+#[\Attribute(\Attribute::IS_REPEATABLE | \Attribute::TARGET_CLASS | \Attribute::TARGET_METHOD)]
```

Comment:
Does it make sense to have multiple `Cache` attributes on the same class/method?